lib/routes/account.js

```diff
@@ -509,13 +511,34 @@ module.exports = function (
           if (shouldSendNewDeviceLoginEmail) {
             // The response doesn't have to wait for this,
             // so we don't return the promise.
-            mailer.sendNewDeviceLoginNotification(
-              emailRecord.email,
-              userAgent.call({
-                acceptLanguage: request.app.acceptLanguage,
-                timestamp: Date.now()
-              }, request.headers['user-agent'])
-            )
+            var location = null
+            var timeZone = null
+            geodb(ip)
```

Comment:
Good catch @rfk, your suggestion is probably the best route to go. You can probably add the helper [here](https://github.com/mozilla/fxa-auth-server/blob/master/lib/routes/utils/request_helper.js).